requests_futures/sessions.py

```diff
@@ -33,9 +40,12 @@ def __init__(self, executor=None, max_workers=2, session=None, *args,
 
         Notes
         ~~~~~
+        * `ProcessPoolExecutor` may be used with Python > 3.3.5;
+          however, if `background_callback` function is given, it must be
+          pickle-able (e.g importable or top level in module).
 
-        * ProcessPoolExecutor is not supported b/c Response objects are
-          not picklable.
+          Additionally, if on Python < 3.5 an existing session object MUST be
+          passed when initializing with `ProcessPoolExecutor`.
```

Comment:
What will the error message/failure case be if someone fails to do so? If it's non-obvious it'd be good to check for that case and throw a better one.